### DISCUSSION TITLE
去除logo

### DISCUSSION THREAD

**Discussion by ChenyangRan** [sjtug/SJTUThesis#1008] [提问] [answered]
写论文时候希望能够去除第一页的logo ![image](https://github.com/sjtug/SJTUThesis/assets/29306735/0269e096-817f-43bf-8bc7-095059b4ccb9)

同时将其改为
![image](https://github.com/sjtug/SJTUThesis/assets/29306735/9c2909a1-2ba7-47f0-90d3-febf9ee7f376)

修改了类型并没有用。
![image](https://github.com/sjtug/SJTUThesis/assets/29306735/24bf52ed-27b5-4e79-aef9-ff1f17406c2c)

**Comment by LogCreative** [comment, marked as answer]
你可能正在看老版的论文模板，如有需要，请使用 v1 版本的模板。
